drivers/gpio/gpio_intel_apl.c

```diff
@@ -243,7 +243,7 @@ static int gpio_intel_apl_config(struct device *dev, int access_op,
 	/* setup interrupt if desired */
 	if (flags & GPIO_INT) {
 		/* invert signal for interrupt controller */
-		if (flags & GPIO_INT_ACTIVE_LOW) {
+		if ((flags & GPIO_INT_ACTIVE_LOW) == GPIO_INT_ACTIVE_LOW) {
```

Comment:
So... `if(flags & 0) == 0)` ...?   That doesn't seem right either. :)

Surely there should be separate macros for the "GPIO_INT_ACTIVITY" field mask vs. the "LOW/HI" enumerants, no?

Reply:
Ah... shouldn't have written code during meetings... :P

Fixed it in new push.